draft-ietf-core-sid-latest.md

```diff
@@ -555,18 +550,16 @@ This document registers one YANG module in the "YANG Module Names" registry {{RF
 * name:         ietf-sid-file
 * namespace:    urn:ietf:params:xml:ns:yang:ietf-sid-file
 * prefix:       sid
-* reference:    [[THISRFC]]
+* reference:    [[RFC XXXX]]
 
-## CoAP Content-Formats Registry
-
-This document adds the following Content-Format to the "CoAP Content-Formats", within the "Constrained RESTful Environments (CoRE) Parameters" registry.
+// RFC Ed.: please replace XXXX with RFC number and remove this note
 
-| Media Type                            | Content Coding | ID   | Reference |
-| application/yang-data+cbor; id=sid    |                | TBD1 | RFC XXXX  |
-{: align="left"}
+## CoAP Content-Formats Registry
 
-// RFC Ed.: replace TBD1 with assigned IDs and remove this note.
-// RFC Ed.: replace RFC XXXX with this RFC number and remove this note.
+A Content-Format for `application/yang-data+cbor; id=sid` has been
+added to the "CoAP Content-Formats" within the "Constrained RESTful
+Environments (CoRE) Parameters" registry, in
+{{I-D.ietf-core-yang-cbor}}.
```

Comment:
It seems this section is no longer needed then?

Reply:
It is not needed.  However, it is useful for people who wonder about Content-Format registrations.

Reply:
If IANA is happy with a no-op section, I'm happy too 😊